src/main/java/com/citusdata/migration/datamodel/TableIndex.java

```diff
@@ -6,12 +6,12 @@
 
 	public final String tableName;
 	public final String name;
-	
+
 	private final List<String> columnNames;
 
 	public TableIndex(String tableName, String name, List<String> columnNames) {
 		this.tableName = tableName;
-		this.name = name;
+		this.name = String.join("-", this.tableName, name);
```

Comment:
1) using `-` is a bit unusual in PostgreSQL land because it requires quoting. Maybe `_` ?
2) table and index names in PostgreSQL are limited to 63 characters, so this might cause other failures

Reply:
1. Ah, good point. I can update that.
2. Yeah I didn't run into that but figured it might be the case. We could truncate the name to 63 characters. There would still be a chance of duplicate names but probably a much smaller one.